students.rb

```diff
@@ -1,4 +1,4 @@
-require './person'
+require './persons'
```

Comment:
- [ ] Good job working on this but please ensure that there is a method that serves as the 'belongs-to' `(a student belongs to a classroom)`.
- [ ] Also please kindly ensure that there is a getter for the classroom.
`attr_reader :classroom`